plugins/org.eclipse.n4js/src/org/eclipse/n4js/resource/PostProcessingAwareResource.java

```diff
@@ -125,14 +130,14 @@ public Throwable getPostProcessingThrowable() {
 
 	@Override
 	public void discardDerivedState() {
+		discardPostProcessingResult();
```

Comment:
Should `discardPostProcessingResult();` be called after `super.discardDerivedState();`?

Reply:
Good point. I was thinking about the exact same thing, as well. Esp. because the old code cleared flag `fullyPostProcessed` after the super call.

However, because the creation order is:
* create derived state
* create post-processing result

I think the better deletion order is:

* delete post-processing result
* delete derived state

(though this is a (very minor) modification of the old behavior)